R/RLearner_regr_gbm.R

```diff
@@ -30,10 +32,39 @@ trainLearner.regr.gbm = function(.learner, .task, .subset, .weights = NULL, ...)
   f = getTaskFormula(.task)
   if (is.null(.weights)) {
     f = getTaskFormula(.task)
-    gbm::gbm(f, data = getTaskData(.task, .subset), ...)
+
+    params <- list(...)
+    if("alpha" %in% names(params)){
+      alpha <- params$alpha
+      params$alpha <- NULL
+    } else {
+      alpha <- 0.5
+    }
+    if(params$distribution %in% "quantile"){
+      params$distribution <- list(name = "quantile", alpha = alpha)
+    }
+    params$formula <- f
+    params$data <- getTaskData(.task, .subset)
+
+    do.call(gbm::gbm, params)
   } else {
     f = getTaskFormula(.task)
-    gbm::gbm(f, data = getTaskData(.task, .subset), weights = .weights, ...)
+
+    params <- list(...)
+    if("alpha" %in% names(params)){
+      alpha <- params$alpha
+      params$alpha <- NULL
+    } else {
+      alpha <- 0.5
+    }
+    if(params$distribution %in% "quantile"){
+      params$distribution <- list(name = "quantile", alpha = alpha)
+    }
+    params$formula <- f
+    params$data <- getTaskData(.task, .subset)
+    params$weights <- .weights
+
+    do.call(gbm::gbm, params)
   }
 }
 
```

Comment:
Can you also please change the test to test this new parameter value?

Reply:
Done